Makefile

```diff
@@ -46,6 +46,9 @@ run-debug: build-debug
 	bash ./scripts/ci/deploy/databend-query-standalone.sh
 
 build:
+	bash ./scripts/build/build-debug.sh
```

Comment:
Some docker use `make build` as release build, also need change to `make build-release`

Reply:
Resolved